go/oasis-test-runner/scenario/e2e/runtime/txsource.go

```diff
@@ -418,25 +419,27 @@ func (sc *txSourceImpl) Fixture() (*oasis.NetworkFixture, error) {
 
 	// Storage committee.
 	f.Runtimes[1].Storage.GroupSize = uint64(sc.numStorageNodes)
-	f.Runtimes[1].Storage.MinPoolSize = f.Runtimes[1].Storage.GroupSize
+	f.Runtimes[1].Constraints[scheduler.KindStorage][scheduler.RoleWorker].MinPoolSize.Limit = uint16(sc.numStorageNodes)
```

Comment:
kinda makes it awkward that we used uint64 for the group size :thinking:

Reply:
Yes, I think it makes no sense. Should we just change the types of those as well since we can do breaking changes?

Reply:
Changed those to uint16 as well.

Reply:
sounds good